### PR TITLE
Updating some of the muon dataset and z->mumu cuts

### PR DESCRIPTION
**Pull request by mlj5j**

### CODE REVIEW

File: BackgroundEstimation/python/config_cfg.py
Comment (carriganm95) [inline BackgroundEstimation/python/config_cfg.py:1]: just make this false again so we don't accidentally run it

File: StandardAnalysis/python/config_cfg.py
Comment (carriganm95) [inline StandardAnalysis/python/config_cfg.py:1]: same here just add the comment back

**Review by carriganm95 — COMMENTED** [review]
Looks good just small changes then you can merge

File: BackgroundEstimation/python/config_cfg.py
Comment (mlj5j) [reply BackgroundEstimation/python/config_cfg.py:1]: Done

File: StandardAnalysis/python/config_cfg.py
Comment (mlj5j) [reply StandardAnalysis/python/config_cfg.py:1]: done

**Pull request merged**